internal/app/connectconformance/test_case_library.go

```diff
@@ -410,18 +409,35 @@ func hasRawResponse(reqs []*anypb.Any) bool {
 	return false
 }
 
+// returns the response definition from an Any representing a unary request.
```

Comment:
nit: Go doc convention is to always start with the element name.
```suggestion
// getUnaryResponseDefinition returns the response definition from an Any representing a unary request.
```